### PR TITLE
Alternative maxquant 

### PR DESCRIPTION
**Pull request by KennethVerheggen**
Please verify this version meets proposed changes

### CODE REVIEW

File: src/maxquant/maxquant/script.py [hunk 1]
```diff
@@ -80,522 +89,27 @@
...
+                  output=par['output'],
+                  fastas=zip(par['reference'],par['ref_taxonomy_id']),
+                  experiments=experiment_names,
+                  match_between_runs=par['match_between_runs'],
+                  match_between_runs_settings=tsv_dispatcher['match_between_runs_settings'],
+                  ms_instrument_settings=tsv_dispatcher['ms_instrument_settings'],
+                  group_type_settings=tsv_dispatcher['group_type_settings'],
+                  quantMode=par['quantMode'],
+                  mainSearchMaxCombinations=par['mainSearchMaxCombinations']
+                 )
```
Comment (rcannood) [inline src/maxquant/maxquant/script.py:107]: fix indentation

File: src/maxquant/maxquant/templates/root.xml.jinja [hunk 2]
```diff
@@ -0,0 +1,520 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MaxQuantParams xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+   <fastaFiles>
+    {%- for path,taxid in fastas -%}
+        {% block fastafileinfo scoped required %}{% endblock %}
```
Comment (rcannood) [inline src/maxquant/maxquant/templates/root.xml.jinja:5]: It's a nice proof of concept but I don't think using a block with a second file is needed at this stage. I suggest simply copying the contents of the block into the `root.xml.jinja`.

@DriesSchaumont @Kenneth-DI What do you think?

Comment (rcannood) [inline src/maxquant/maxquant/templates/root.xml.jinja:5]: It's a nice proof of concept but I don't think using a block with a second file is needed at this stage. I suggest simply copying the contents of the block into the `root.xml.jinja`.

@DriesSchaumont @KennethVerheggen What do you think?

File: src/maxquant/maxquant/script.py (hunk 1 above)
Comment (rcannood) [inline src/maxquant/maxquant/script.py:107]: fix indentation of the `)`

**Review by rcannood — CHANGES_REQUESTED** [review]
This looks so much better :heart_eyes: 

@KennethVerheggen Could you merge the main branch into this branch so that it can be merged back into main?

File: src/maxquant/maxquant/templates/root.xml.jinja (hunk 2 above)
Comment (DriesSchaumont) [reply src/maxquant/maxquant/templates/root.xml.jinja:5]: Sure, lets do that. Opening a second file to see the contents is a pain. Could be useful in the future where we need to add different xml blocks depending on parameters. But for now lets not split files

Comment (KennethVerheggen) [reply src/maxquant/maxquant/templates/root.xml.jinja:5]: Will replace the extra file and integrally put the block in the root xml !

**Comment by DriesSchaumont** [comment]
LGTM  @rcannood

**Review by DriesSchaumont — APPROVED** [review]

**Pull request merged**